airflow/contrib/operators/swarm_operator.py

```diff
@@ -0,0 +1,146 @@
+'''
+Run ephemeral Docker Swarm services
+'''
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from docker import types
+
+from airflow.exceptions import AirflowException
+from airflow.operators.docker_operator import DockerOperator
+from airflow.utils.decorators import apply_defaults
+from airflow.utils.strings import get_random_string
+
+
+class SwarmOperator(DockerOperator):
```

Comment:
I think, the better name is `DockerSwarmExecuteCommand` or something similar. WDYT?

Reply:
I agree `Swarm` is not as popular as `Docker`, so prepending `Docker` makes sense to me.

This Operator works in the same fashion as the existing `DockerOperator` in Airflow - since that doesn't have `ExecuteCommand` in its name, I don't see the need to add it here either. (Or am I missing your point?)

So WDYT about `DockerSwarmOperator`?

Reply:
I think that the name of this Docker's operator should be changed. I think that the name `DockerSwarmOperator` may stay for now. 

I will suggest changing names as parts of AIP:
https://cwiki.apache.org/confluence/display/AIRFLOW/AIP-21%3A+Changes+in+import+paths

Reply:
Sure, I'll change the name to `DockerSwarmOperator`. Thank you for your inputs.